# Disorder
A ransomware made in python.

This ransomware was created for fun, not for malicious use.
This means that the creator of this virus (Me) is not responsible for any damage done by this virus. Use it at your own risk.
Also to test this virus you need to turn on the internet.
There is curently no decryptor.
